browser/brave_app_controller_mac.h

```diff
@@ -6,14 +6,24 @@
 #ifndef BRAVE_BROWSER_BRAVE_APP_CONTROLLER_MAC_H_
 #define BRAVE_BROWSER_BRAVE_APP_CONTROLLER_MAC_H_
 
+#import <Cocoa/Cocoa.h>
+
 #import "chrome/browser/app_controller_mac.h"
 
 // Manages logic to switch hotkey between copy and copy clean link item.
-@interface BraveAppController : AppController {
-  NSMenuItem* _copyMenuItem;
-  NSMenuItem* _copyCleanLinkMenuItem;
-}
+@interface BraveAppController : AppController
+
+@property(readonly, nonatomic, class) BraveAppController* sharedController;
+
+@property(nonatomic, assign) NSMenuItem* copyMenuItem;
+@property(nonatomic, assign) NSMenuItem* copyCleanLinkMenuItem;
+
+// NOLINTNEXTLINE
+- (NSMenuItem*)copyMenuItem __attribute__((objc_method_family(none)));
+// NOLINTNEXTLINE
+- (NSMenuItem*)copyCleanLinkMenuItem __attribute__((objc_method_family(none)));
```

Comment:
Assuming the associated properties can be changed to `strong` then these won't be needed

Reply:
cc @sangwoo108